schema/site_metadata.json

```diff
@@ -50,6 +58,59 @@
             "type": "number"
         }
       }
+    },
+    "area": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "m2": {
+            "description": "Area of the site in square meters",
+            "type": "number",
+            "examples": [1234, 256, 998765]
+        }
+      }
+    },
+    "source": {
```

Comment:
"source" doesn't feel right here -- can't remember our conversation about it. Really the bucket seems to be about "external links to places where information is stored" -- and could be used (e.g.) as an _output_ for some of the tools about the side (so definitely not source). Does something like "externals" or "links" make more sense?

e.g. consider the use-case of "where running the registrar tool should output a log of what it does so it can be captured in (e.g.) a Google sheet?"